src/config/session-management.md

```diff
@@ -0,0 +1,32 @@
+# Session and Seat Management
+
+Session and seat management is not necessary for every setup, but it can be used
+to safely create temporary runtime directories, provide access to hardware
+devices, and control system shutdown.
+
+## D-Bus
+
+D-Bus is an IPC (inter-process communication) mechanism used by userspace
+software in Linux. There are two uses for it: as a system bus, and as a user
+bus. To use it as a system bus, you should [enable](./services/index.md) the
+`dbus` service. To use it as a user bus, you can start a given program (usually
+a window manager or interactive shell) with
+[dbus-run-session(1)](https://man.voidlinux.org/dbus-run-session.1). Most
+desktop environments, if launched through an adequate display manager, will
+launch a D-Bus session themselves.
+
+## elogind
+
+[elogind(8)](https://man.voidlinux.org/elogind.8) is a standalone version of
+`systemd-logind`, a service to manage user logins. This service provides
+necessary features for most desktop environments and Wayland compositors. To
+make use of its features, install the `elogind` package and enable the `dbus`
+service. This might require a reboot.
```

Comment:
All you need to do is restart your session AFAIK (ie. go back to `login`). When you authenticate, `elogind` will start.

Reply:
I was talking more specifically about dbus itself. That's what @flexibeast experienced, at least, IIRC. `elogind` requires only that you log out and then log back in?

Reply:
It seems to depend on the context. In my VM testing, i've found that GNOME requires a reboot after enabling `dbus`, but KDE doesn't. i don't recall anything needing to be done after installing `elogind`.

Reply:
(Oh and i should add: logging out and back in again after enabling `dbus` is not enough for GNOME. i just checked this a couple of hours ago, and it definitely needs a reboot.)